register.lua

```diff
@@ -1,104 +1,264 @@
+-- STONE --------------------------------------------------------------------------------
 
 mymillwork.register_all(
 	"default_stone",
 	"Stone",
 	"default_stone.png",
-	{cracky = 3,not_in_creative_inventory=1},
+	{cracky = 3, not_in_creative_inventory=1},
 	"default:stone"
 	)
 
 mymillwork.register_all(
 	"default_stone_brick",
 	"Stone Brick",
 	"default_stone_brick.png",
-	{cracky = 3,not_in_creative_inventory=1},
+	{cracky = 3, not_in_creative_inventory=1},
 	"default:stonebrick"
 	)
 
+mymillwork.register_all(
+	"default_desert_stone",
+	"Desert Stone",
+	"default_desert_stone.png",
+	{cracky = 3, not_in_creative_inventory=1},
+	"default:desert_stone"
+	)
+
+mymillwork.register_all(
+	"default_desert_stonebrick",
+	"Desert Stone Brick",
+	"default_desert_stone_brick.png",
+	{cracky = 3, not_in_creative_inventory=1},
+	"default:desert_stonebrick"
+	)
+
 mymillwork.register_all(
 	"default_cobble",
 	"Cobble",
 	"default_cobble.png",
-	{cracky = 3,not_in_creative_inventory=1},
+	{cracky = 3, not_in_creative_inventory=1},
 	"default:cobble"
 	)
 
 mymillwork.register_all(
-	"default_desert_stone",
-	"Desert Stone",
-	"default_desert_stone.png",
-	{cracky = 3,not_in_creative_inventory=1},
-	"default:desert_stone"
+	"default_mossycobble",
+	"Mossy Cobble",
+	"default_mossycobble.png",
+	{cracky = 3, not_in_creative_inventory=1},
+	"default:mossycobble"
 	)
 
+-- SANDSTONE -----------------------------------------------------------------------------
+
 mymillwork.register_all(
 	"default_sandstone",
 	"Sandstone",
 	"default_sandstone.png",
-	{crumbly=2, cracky = 3, not_in_creative_inventory=1},
+	{crumbly=2,  cracky = 3,  not_in_creative_inventory=1},
```

Comment:
Replacement issue. Please replace the double space with a single space.

Reply:
Right... I wanted to fix your lack of whitespaces around "=" and after "," and used a wrong regex...